tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -467,27 +499,30 @@ def sync_volumes(volumes, index, vol_options):
             try:
                 sbs = NS._int.client.read(
                     "clusters/%s/Bricks/all/"
-                    "%s/status" % (
+                    "%s/%s/status" % (
                         NS.tendrl_context.
                         integration_id,
-                        brick_name
+                        NS.node_context.fqdn,
+                        brick_name.replace(
```

Comment:
@nnDarshan Is the brick name correctly derived here? As per our discussion on not having "/" for any attribute value

Reply:
@shtripat @r0h4n This is technically fine, but `brick_name.split(":_")[-1]` would suffice now.

Reply:
Rabsed with latest master below and should be fine